javascripts/content_scripts/privly.js

```diff
@@ -378,10 +372,11 @@ var privly = {
        };
 
     //Styling and display attributes
-     for(var key in attrs) 
-     {
-       iFrame.setAttribute(key, attrs[key]);
-     }
+    for(var key in attrs) {
+	    if (iFrame) {
+		    iFrame.setAttribute(key, attrs[key]);
```

Comment:
Why are you testing for the existence of the iframe? It was created a few lines earlier.